Pod/Classes/UIApplication+Rx.swift

```diff
@@ -54,14 +54,36 @@ public func ==(lhs: AppState, rhs: AppState) -> Bool {
     }
 }
 
+public struct RxAppState {
+    /**
+     Allows for the app version to be stored by default in the main bundle from `CFBundleShortVersionString` or
+     a custom implementation per app.
+     */
+    public static var getCurrentAppVersion: ((Void) -> String?) = { _ in return Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String}
```

Comment:
Agreed on the renaming.

As for having it as a function, I was messing around with the version being calculated from a bunch of things. But thinking on it, that can still be done e.g. `RxAppState.currentAppVersion = someFunctionThatReturnsAString`

I'll make it a String.